snmp/datadog_checks/snmp/data/profiles/cisco-catalyst.yaml

```diff
@@ -268,9 +265,6 @@ sysobjectid:
   - 1.3.6.1.4.1.9.1.1328
   - 1.3.6.1.4.1.9.1.1329
   - 1.3.6.1.4.1.9.1.1330
-  - 1.3.6.1.4.1.9.1.1334
-  - 1.3.6.1.4.1.9.1.1335
-  - 1.3.6.1.4.1.9.1.1336
```

Comment:
Can you add a note in PR description about moving those sysobjectid(s) ?

Reply:
Sure

Reply:
Done!